structuredevent-service-legacy/src/main/java/com/sequenceiq/cloudbreak/structuredevent/service/telemetry/mapper/ClusterUseCaseMapper.java

```diff
@@ -98,6 +102,10 @@ private UsageProto.CDPClusterStatus.Value lastStepToUseCaseMapping(String rootFl
                     useCase = getClusterStatus(nextFlowState, "FLOWCHAIN_FINALIZE_FINISHED_STATE",
                             UsageProto.CDPClusterStatus.Value.DOWNSCALE_FINISHED, UsageProto.CDPClusterStatus.Value.DOWNSCALE_FAILED);
                     break;
+                case "StopStartUpscaleFlowEventChainFactory":
+                    useCase = getClusterStatus(nextFlowState, "STOPSTART_UPSCALE_FINALIZE_STATE",
+                            UsageProto.CDPClusterStatus.Value.STOP_START_DOWNSCALE_FINISHED, UsageProto.CDPClusterStatus.Value.STOP_START_DOWNSCALE_FAILED);
```

Comment:
This should be the UPSCALE protos?

And, StopStartDownscaleEventChainFactory missing ?

Reply:
Sorry about this, misinterpreted the method. Updated the protos and added the `StopStartDownscaleFlowEventChainFactory`